README.md

```diff
@@ -225,48 +241,76 @@ These instructions were tested on Windows Server 2016 and Windows 11 machines, t
     (wait for it to install)
     quit
     ```
-6. Extract the latest release of NT;RE into `<YOUR_LOCATION>\ntrebuild`, so you will have a directory `<YOUR_LOCATION>\ntrebuild\neo` with a `gameinfo.txt` inside.
-7. Allow all Inbound and Outbound TCP and UDP requests for the following ports via Windows Firewall. [See how](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/configure#create-an-inbound-port-rule)
+6. The distribution of the dedicated servers currently misplaced some few files, here's the following files missing on Windows:
+    ```
+    srcds_win64.exe
+    ```
+    You may either fetch them from the Windows Source SDK 2013 Multiplayer client (appid 243750) or TF2's Dedicated Server (appid 232250).
+    Once they're fetched, place them in the `<YOUR_LOCATION>\ntrebuild` directory.
+
+7. Extract the latest release of NT;RE into `<YOUR_LOCATION>\ntrebuild`, so you will have a directory `<YOUR_LOCATION>\ntrebuild\neo` with a `gameinfo.txt` inside.
+8. Allow all Inbound and Outbound TCP and UDP requests for the following ports via Windows Firewall. [See how](https://learn.microsoft.com/en-us/windows/security/operating-system-security/network-security/windows-firewall/configure#create-an-inbound-port-rule)
     * 27015 TCP+UDP (you can keep the TCP port closed if you don't need RCON support)
     * 27020 UDP
     * 27005 UDP
     * 26900 UDP
-8. Your server should be ready to go, launch it inside your main directory (`(...)\ntrebuild\`) with the following command: (You can alter any argument to your liking, except `-game` and `-neopath`) 
+9. Your server should be ready to go, launch it inside your main directory (`(...)\ntrebuild\`) with the following command: (You can alter any argument to your liking, except `-game` and `-neopath`) 
 ```
-srcds.exe -game neo -neopath "..\ognt\NeotokyoSource" +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
+srcds_win64.exe -game neo -neopath "..\ognt\NeotokyoSource" +ip <YOUR_IP> -maxplayers <1-32> +map <MAP_NAME>
 ```
 
 ### Testers/Devs
+**NOTE: As of the TF2-SDK update, Linux will fail to find the files necessary to startup a working server. Just follow the normal dedicated server instruction for now.**
+
```

Comment:
Could use the [alerts syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) for this note, not that important though